### PR TITLE
Refactor: Makefile

### PR DESCRIPTION
**Pull request by Mohiiit**
# Description

This PR includes several improvements to our build and configuration systems:

1. Updated bootstraper dependency in the Makefile to the latest version
2. Enhanced the update-json script functionality and now it accepts any JSON file as input (previously limited to fixed file)
3. Fixed config hash issues through the updated bootstraper

### CODE REVIEW

**Review by apoorvsadana — APPROVED** [review]
lgtm

File: bootstrap_input.json
Comment (ocdbytes) [inline bootstrap_input.json:1]: can we add this file to .gitignore ? Or is this a design choice ?

**Review by ocdbytes — COMMENTED** [review]
reviewed. Everything else looks good to me.

File: bootstrap_input.json
Comment (Mohiiit) [reply bootstrap_input.json:1]: plan was to have a file for user to use, makes things easy

Comment (ocdbytes) [reply bootstrap_input.json:1]: cool

**Review by ocdbytes — APPROVED** [review]

**Pull request merged**